development/main.postgressql.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_ssm_parameter" "db_username" {
+  name = "/housing-finance/development/db-username"
+}
+data "aws_ssm_parameter" "db_password" {
+  name = "/housing-finance/development/db-password"
+}
+
+# HFS Postgres Master database
+module "postgres_db_master" {
+  source               = "../modules/postgres"
+  environment_name     = var.environment_name
+  db_identifier        = "housing-finance-master"
+  db_engine            = "postgres"
+  db_engine_version    = "16.1"
+  db_instance_class    = "db.t3.xlarge"
+  db_name              = "hfs-postgres-master"
+  # TODO: Replace these with a parameter group
+  db_username          = data.aws_ssm_parameter.db_username.value
+  db_password          = data.aws_ssm_parameter.db_password.value
+  vpc_id               = "vpc-0d15f152935c8716f"
+  db_allocated_storage = 1200
+  db_port              = "5432"
+  subnet_ids           = ["subnet-05ce390ba88c42bfd","subnet-0140d06fb84fdb547"]
+  storage_encrypted    = true
+  multi_az             = false
+  enabled_cloudwatch_logs_exports = ["postgresql"]
```

Comment:
Is this the name of the log group that will be written to?
If so, perhaps, we could make it more specific for it be more searchable?

Reply:
It's one of the available log types: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster